keychain/tests/test_keychain_backend.py

```diff
@@ -57,7 +57,7 @@ def test_keychain_bakend(self):
         backend._inverse_keychain()
         account = backend._get_existing_keychain()
         self.assertEqual(
-            account.data, '{"a": "o", "c": "b"}',
+            account.get_data(), {"a": "o", "c": "b"},
```

Comment:
You can use `self.assertDictEqual` instead for not having to worry about the order.